keyboards/hator/gravity/tkl/readme.md

```diff
@@ -0,0 +1,27 @@
+# gravity
+
+![gravity](https://imgur.com/TfBlmHo)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Alex](https://github.com/AlexHtr)
+* Hardware Supported: Wired mode, side led enabled
+* Hardware Availability: https://hator.gg/keyboards/gravity_tkl/
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb hator/gravity/tkl
+
+Flashing example for this keyboard:
+
+    qmk flash -kb hator/gravity/tkl -km via
```

Comment:
`qmk flash` is not a supported feature at this point in time. Did you even test this?